tests/main/snap-discard-ns/mount.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+if [ "$(command -v python3)" != "" ]; then
+	exec ./mount-py3.py "$@";
```

Comment:
Can we have just one version of the script? The way you've written py2 version, it  should work with py3 just fine.

Reply:
I tried to initially but in the end I decided no to. Using one version of the script puts most pressure on the unicode vs string vs bytes across the two versions. Right now both versions are correct but this could be subtly broken with a single script.

Reply:
I think one script is enough. With the exception of `b""`, I don't see anything wrong with py2 version that would blow up under py3.